google-cloud-compute-v1/google-cloud-compute-v1.gemspec

```diff
@@ -23,7 +23,7 @@ Gem::Specification.new do |gem|
 
   gem.required_ruby_version = ">= 2.5"
 
-  gem.add_dependency "gapic-common", "~> 0.4"
+  gem.add_dependency "gapic-common", "~> 0.4", '>=0.4.1'
```

Comment:
Please use double quotes. Also not sure if the space between the operator and version is required, but it's used consistently elsewhere, so please do it here too.

Reply:
fixed